packages/common/src/ui/layout/tables/components/Expand/MiniTable.tsx

```diff
@@ -17,14 +17,16 @@ export const MiniTable = <T extends DomainObject>({
   JSXElementConstructor<MiniTableProps<T>>
 > => {
   return (
-    <Box p={1} style={{ padding: '0 100px', width: '100%' }}>
+    <Box p={1} style={{ padding: '0 100px 5px 100px', width: '100%' }}>
```

Comment:
adds the padding below the row, just to space it out from the next row a little more